contracts/generate_contract_address_table.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2024 RISC Zero, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from urllib.parse import urljoin
+import tomllib
+import sys
+
+def load_toml(file_path):
+    """Load the TOML file from the given path."""
+
+    with open(file_path, "rb") as f:
+        return tomllib.load(f)
+
+class Link:
+    def __init__(self, tag, url):
+        self.tag = tag
+        self.url = url
+
+def generate_table_for_chain(chain_name, chain_data, version):
+    """Generate markdown content for a specific blockchain."""
+
+    chain_id = chain_data['id']
+
+    links = []
+    etherscan_url = chain_data['etherscan-url']
+
+    md_content = f"### {chain_data['name']} ({chain_id})\n\n"
+    md_content += "| Contract                     | Address   |\n"
+    md_content += "| ---------------------------- | --------- |\n"
+
+    # Add timelock controller and router
+    # NOTE: TimelockController and RiscZeroVerifierRouter may not be deployed to all chains.
+    timelock_addr = chain_data['timelock-controller']
+    router_addr = chain_data['router']
+
+    # The router is the contract devs are most likely to interact with, so it is first.
+    md_content += f"| [RiscZeroVerifierRouterRouter][router-src] | [`{router_addr}`][router-{chain_id}-etherscan] |\n"
```

Comment:
```suggestion
    md_content += f"| [RiscZeroVerifierRouter][router-src] | [`{router_addr}`][router-{chain_id}-etherscan] |\n"
```

Reply:
Oh dang, that made it into the docs too 😅 
https://dev.risczero.com/api/blockchain-integration/contracts/verifier

Reply:
170127d99ea39ab0b1df4328df0581ee7f330fa8